v20.2/create-schedule-for-backup.md

```diff
@@ -0,0 +1,222 @@
+
+## Examples
+
+### Create a schedule for full backups only (core)
+
+Core users can only use backup scheduling for full backups:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE SCHEDULE core_schedule_name
+  FOR BACKUP INTO 's3://test/schedule-test-core?AWS_ACCESS_KEY_ID=x&AWS_SECRET_ACCESS_KEY=x'
+    WITH revision_history RECURRING '@daily'
+    FULL BACKUP ALWAYS;
+~~~
+~~~
+     schedule_id     |        name        | status |         first_run         | schedule |                                                                                       backup_stmt
+---------------------+--------------------+--------+---------------------------+----------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  588799238330220545 | core_schedule_name | ACTIVE | 2020-09-11 00:00:00+00:00 | @daily   | BACKUP INTO 's3://test/schedule-test-core?AWS_ACCESS_KEY_ID=x&AWS_SECRET_ACCESS_KEY=x' WITH revision_history, detached
+(1 row)
+~~~
+
+To use the other backup scheduling features, you need an [enterprise license](enterprise-licensing.html).
+
+### Create a scheduled backup for a cluster
+
+This example creates a schedule for a cluster backup with revision history that's taken every day at midnight:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE SCHEDULE schedule_name
+  FOR BACKUP INTO 's3://test/backups/schedule_test?AWS_ACCESS_KEY_ID=x&AWS_SECRET_ACCESS_KEY=x'
+    WITH revision_history
+    RECURRING '@daily';
+~~~
+
+~~~
+     schedule_id     |     name      |                     status                     |            first_run             | schedule |                                                                               backup_stmt
+---------------------+---------------+------------------------------------------------+----------------------------------+----------+---------------------------------------------------------------------------------------------------------------------------------------------------------
+  588796190000218113 | schedule_name | PAUSED: Waiting for initial backup to complete | NULL                             | @daily   | BACKUP INTO LATEST IN 's3://test/schedule-test?AWS_ACCESS_KEY_ID=x&AWS_SECRET_ACCESS_KEY=x' WITH revision_history, detached
+  588796190012702721 | schedule_name | ACTIVE                                         | 2020-09-10 16:52:17.280821+00:00 | @weekly  | BACKUP INTO 's3://test/schedule-test?AWS_ACCESS_KEY_ID=x&AWS_SECRET_ACCESS_KEY=x' WITH revision_history, detached
+(2 rows)
+~~~
+
+Because the [`FULL BACKUP` clause](#full-backup-clause) was not included, CockroachDB also scheduled a full backup to run `@weekly`. This is the default cadence for incremental backups `RECURRING` <= 1 day.
```

Comment:
I believe if you create a cadence of @hourly we default to daily fully backups. Might want to check with yevgeniy on this. Bottom line is, if you don't specify we choose one for you so that it's not incremental forever.